presto-native-execution/presto_cpp/presto_protocol/special/Language.cpp.inc

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Is this needed anymore? I see the protocol update but this file doesn't seem to be used anywhere.
And this code is actually in the presto_protocol.cpp line 1514ff.

Reply:
This change is needed to define custom serialization for `Language`, without this change the following json will be returned:
```
"language": {
  "language": "CPP"
}
```
Instead of the expected json:
```
"language": "CPP"
```
This change was added to fix an error during e2e testing and I believe the code in `presto_protocol.cpp` contains this logic because it is defined in this file, after reverting this the corresponding function in `presto_protocol.cpp` would be:
```
void to_json(json& j, const Language& p) {
  j = json::object();
  to_json_key(j, "language", p.language, "Language", "String", "language");
}
```
@pdabre12 could you please confirm if this is still needed?

Reply:
This change is still needed.